cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/base_outlet.json

```diff
@@ -22,5 +22,6 @@
   "rating-group-id": {
     "office-id": "SPK",
     "name": "Rating-BIGH-TG1"
-  }
+  },
+  "rating-spec-id": "Rating-BIGH-TG1.Elev;Flow-Uncontrolled_Spillway.Linear.Production"
```

Comment:
This should use Elev,Opening-Tainter_Gate;Flow-Uncontrolled_Spillway.  I know it's a bit petty, but a tainter gate is not an uncontrolled spillway.  I don't know if this needs updating - @adamkorynta thoughts?

Reply:
Updated naming, open to suggestions